source/OutfitterPanel.cpp

```diff
@@ -73,8 +73,18 @@ OutfitterPanel::OutfitterPanel(PlayerInfo &player)
 void OutfitterPanel::Step()
 {
 	CheckRefill();
-	DoHelp("outfitter");
 	ShopPanel::Step();
+	if(GetUI()->IsTop(this)) {
+		if(checkedhelp == false) {
+			if(DoHelp("outfitter 1") == true) {
+				checkedhelp = true;
+				} else if(DoHelp("outfitter 2") == true) {
+					checkedhelp = true;
+					} else if(DoHelp("outfitter 3") == true) {
+						checkedhelp = true;
+						}
+		}
+	}
```

Comment:
At minimum, fix the indentations and remove the verbose boolean comparisons.

```suggestion
	if(GetUI()->IsTop(this) && !checkedHelp) {
		if(!DoHelp("outfitter 1") && !DoHelp("outfitter 2") && !DoHelp("outfitter 3")) {
			// All help messages have now been displayed.
			checkedHelp = true;
		}
	}
```
This suggestion will make the dialogs show in succession for a single visit. I'm not sure it makes sense for the dialogs to appear on separate visits, given their current phrasing.

Reply:
I was thinking that players probably don't need to be bombarded by three successive boxes all at once - I thought it better to break it up over a few successive visits. But I suppose it could be right together too.